### PR TITLE
docs: show hamburger menu for getting started pages on desktop

### PR DESCRIPTION
**Pull request by malladi-s**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ x ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ x ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: [29773](https://github.com/angular/angular/issues/29773)

## What is the new behavior?
Fixed a regression where the hamburger was shown for desktops when the user is in any of the getting started pages.

## Does this PR introduce a breaking change?

- [ ] Yes
- [ x ] No

### CODE REVIEW

File: aio/src/styles/1-layouts/_top-menu.scss [hunk 1]
```diff
@@ -46,6 +46,7 @@ aio-shell.page-resources mat-toolbar.mat-toolbar {
 }
 
 // DOCS PAGES OVERRIDE: HAMBURGER
+aio-shell.folder-getting-started mat-toolbar.mat-toolbar,
```
Comment (brandonroberts) [inline aio/src/styles/1-layouts/_top-menu.scss]: Move below the `folder-docs` line so they remain in order.

**Review by brandonroberts — CHANGES_REQUESTED** [review]
Amend git commit message to `docs: show hamburger menu for getting started pages on desktop`

**Comment by brandonroberts** [comment]
Thanks for the PR @invictusmaneo! I left you some feedback. If you need any help or its unclear, let me know.

File: aio/src/styles/1-layouts/_top-menu.scss (hunk 1 above)
Comment (invictusmaneo) [reply aio/src/styles/1-layouts/_top-menu.scss]: Oh! alphabetical order. I wrote it in the beginning as getting started the first item in the sidebar.

**Review by brandonroberts — APPROVED** [review]

**Pull request closed without merging**